src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -841,7 +841,10 @@ private static void addFunctionToSchemaMutation(UDFunction function, Mutation.Si
                .add("language", function.language())
                .add("return_type", function.returnType().asCQL3Type().toString())
                .add("called_on_null_input", function.isCalledOnNullInput())
-               .add("argument_names", function.argNames().stream().map((c) -> bbToString(c.bytes)).collect(toList()));
+               .add("argument_names", function.argNames().stream().map((c) -> bbToString(c.bytes)).collect(toList()))
+               .add("deterministic", false)
+               .add("monotonic", false)
+               .add("monotonic_on", Collections.emptyList());
```

Comment:
Nit: we might want to add a comment mentioning why we need this

Reply:
Good idea, I have added an inline comment with a small explanation.